docs/en/ingest-management/fleet-agent-proxy-support.asciidoc

```diff
@@ -132,6 +132,8 @@ HTTP_PROXY=http://my.proxy:8080
 
 After adding environment variables, restart the service.
 
+NOTE: If you use a proxy server to download artifacts from `artifacts.elastic.co`, configure <<fleet-agent-binary-download-settings>>.
```

Comment:
```suggestion
NOTE: If you use a proxy server to download artifacts from `artifacts.elastic.co`, configure <<fleet-agent-binary-download-settings>>. `artifacts.elastic.co` is used to download new versions of the agent when upgrading.
```

LGTM, but I think we should mention that artifacts.elastic.co is required to download upgrades so users know why they would need to configure this.

Feel free to reword this.

Reply:
Actually this might be handled above with `Download artifacts from `artifacts.elastic.co` for subprocesses or binary upgrades` but it is a bit far away from this note